sdcm/tester.py

```diff
@@ -2491,13 +2492,17 @@ def check_latency_during_ops(self):
                                                                           'email_recipients'),
                                                                       events=get_events_grouped_by_category(
                                                                           _registry=self.events_processes_registry))
-        if self.db_cluster.latency_results and self.create_stats:
-            self.db_cluster.latency_results = calculate_latency(self.db_cluster.latency_results)
-            self.log.debug('collected latency values are: %s', self.db_cluster.latency_results)
-            self.update({"latency_during_ops": self.db_cluster.latency_results})
-
-            self.update_test_details()
-            results_analyzer.check_regression(test_id=self._test_id, data=self.db_cluster.latency_results)
+        latency_results_file = 'latency_results.json'
+        if os.path.exists(latency_results_file):
+            with open(latency_results_file, 'r') as file:
+                latency_results = json.load(file)
```

Comment:
i think better to have single method (static) which could be used everywhere, and name of file should be configurable